client/components/notice/index.jsx

```diff
@@ -98,6 +102,7 @@ export class Notice extends Component {
 
 		return (
 			<div className={ classes }>
+				<NoticeAnnouncement { ...this.props } />
```

Comment:
Just a suggestion, but consider picking out the props that `NoticeAnnouncement` supports instead of passing everything. It'll make it a bit clearer which properties are in use when someone needs to refactor notices down the road.